### PR TITLE
Homepage UI improvements

### LINKED ISSUE #36: Home Page UI improvements
1. Improve Home Page interface
https://cdn.discordapp.com/attachments/802949481552150548/802972305839226910/unknown.png
Background idea:
Gradient - white circle inside and then darkgreen

2. Buttons should change bottom border color after hover (only suggestion) 
https://cdn.discordapp.com/attachments/802949481552150548/802972938029760512/unknown.png
Button examples: (suggestion, for now it just looks bad) 
https://codepen.io/topic/buttons/picks

3. Also change GLOBAL font. You Can use google fonts.

### PR DESCRIPTION
**Pull request by GoraJakub**
Please review

# Homepage startscreen
image

# Button hover
image

# Dropdown list + hover
image

# People 
image

# Dwarfs
image

### CODE REVIEW

File: src/app/app.ts [hunk 1]
```diff
@@ -1,14 +1,10 @@
-
 import { playAudio } from './utils'
 import { GamePage, HomePage } from './screens'
 
 export class App {
   start() {
     console.log('Starting application...')
-    playAudio(0.7);
-    const homepage = new HomePage()
-    const gamepage = new GamePage()
```
Comment (pdybowski) [inline src/app/app.ts]: Restore it. We need this game page for tests for now

**Review by pdybowski — CHANGES_REQUESTED** [review]
Change button border bottom color when hover E.g. to dark green but not too dark.
Remove box shadow from drop down list or just leave border shadow (remove shadow between options). Also add something to highlight current option. So when people are selected and you open drop down "People" (box on the very top) should look different then other options in drop down

**Comment by pdybowski** [comment]
Moreover you can change background color from green to gray when dwarfs are picked and to yellow or some other color that is associated to the people's image when people class is selected

File: src/app/screens/homepage/index.ts [hunk 2]
```diff
@@ -35,16 +53,22 @@ export class HomePage {
...
+    this.startScreen.classList.add(`start_screen--${value}`)
   }
 
-  private handleLogoChange(): void {
-    this.characterButton.addEventListener('change', () => {
-      const value = this.characterButton.value
-      this.removeLogoClasses()
-      this.changeLogo(value)
+  private handleThemeChange(): void {
+    this.characterButtonOptions.forEach((e) => {
```
Comment (pdybowski) [inline src/app/screens/homepage/index.ts]: e is associated with events. you can pass opt or option instead of e

File: src/index.html [hunk 3]
```diff
@@ -3,35 +3,66 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap"
```
Comment (pdybowski) [inline src/index.html]: take this content and place it in assets/fonts/google-fonts.css file

File: src/index.html [hunk 4]
```diff
@@ -3,35 +3,66 @@
...
+      rel="stylesheet"
+    />
     <title></title>
-    
   </head>
   <body>
-
     <audio autoplay loop>
-      <source src="assets/audio/music.mp3" type="audio/mpeg">
+      <source src="assets/audio/music.mp3" type="audio/mpeg" />
```
Comment (pdybowski) [inline src/index.html:10]: we dont need this '/' at the end

File: src/index.html [hunk 5]
```diff
@@ -3,35 +3,66 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
```
Comment (pdybowski) [inline src/index.html]: we dont need it here

**Review by pdybowski — CHANGES_REQUESTED** [review]
remove people.jpg, dwarfs.jpg, elfs.jpg

**Comment by GoraJakub** [comment]
# NEW PHOTOS

## Button hover
image

## Dropdown list + hover
image

## People
image

## Dwarfs
image

File: src/assets/fonts/google-fonts.css [hunk 6]
```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap');
```
Comment (pdybowski) [inline src/assets/fonts/google-fonts.css]: Wejdx ten link, skopiuj keto zawartosc i umiesc go zamiast Teton imports. Dzięki temu tekst będzie zmieniony nawet jak nie będzie połączenia z internetem

**Review by pdybowski — CHANGES_REQUESTED** [review]
Fix Google fonts

**Review by hoolek77 — APPROVED** [review]

**Review by pdybowski — APPROVED** [review]

File: src/app/screens/homepage/index.ts [hunk 7]
```diff
@@ -15,15 +18,37 @@ export class HomePage {
...
+      this.startScreenLogo.classList.remove(`start__screen__logo--${category}`)
+      this.startScreenLogoBackground.classList.remove(`${category}__background`)
+      this.startScreenBackground.classList.remove(`start_screen--${category}`)
+    }
+    this.characterButtonOptions.forEach((e) => {
+      e.classList.remove('start__screen__button--option--selected')
+    })
+  }
+
+  private changeButtonTextContext(value: string) {
```
Comment (bartoszbialecki) [inline src/app/screens/homepage/index.ts]: Typo: TextContent

**Review by bartoszbialecki — APPROVED** [review]

**Review by aleksandraj036 — APPROVED** [review]

**Pull request merged**